src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -23,6 +23,7 @@
 public class PathPlannerCommand extends SequentialCommandGroup {
   
   Alliance alliance;
+  boolean m_perpetual;
```

Comment:
does this need to be a member variable? Probably alliance can be removed as well